src/libs/actions/BankAccounts.js

```diff
@@ -40,6 +42,15 @@ function clearPlaid() {
     Onyx.set(ONYXKEYS.PLAID_LINK_TOKEN, '');
 }
 
+/** Reimbursement account actively being set up */
+let reimbursementAccountInSetup = {};
+Onyx.connect({
```

Comment:
Why do you need this instead of just calling `store.getReimbursementAccountInSetup()`?

Reply:
I removed this in favor of `store.getReimbursementAccountInSetup()`. Thanks!